src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -276,6 +276,13 @@ public void BuildAMassiveApp ()
 				AotAssemblies = true,
 				IsRelease = true,
 			};
+			if (Builder.UseDotNet) {
+				app1.PackageReferences.Clear ();
+				app1.PackageReferences.Add (KnownPackages.XamarinForms_5_0_0_2515);
+				app1.PackageReferences.Add (KnownPackages.XamarinFormsMaps_5_0_0_2515);
+				app1.PackageReferences.Add (KnownPackages.Xamarin_Build_Download_0_11_3);
+
+			}
```

Comment:
Was this test failing, unless we updated Forms?

Reply:
If memory serves is was failing. yes. I can try to back this change out and see what happens.